pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "PowerSploit";
```

`pname`s should be normalized.

```suggestion
  pname = "powersploit";
```